src/Bindings/Validator.ts

```diff
@@ -157,7 +157,7 @@ class DbRowCheck {
     { pointer, errorReporter, arrayExpressionPointer, refs }: ValidationRuntimeOptions
   ) {
     const client = this.database.connection(connection)
-    const query = client.from(table)
+    const query = client.from(table).select(this.database.raw('1'))
```

Comment:
Do we need `this.database.raw(1)` here. Will just `1` not work?. Coz I see in tests you are using just `select(1)` with the Knex query builder

Reply:
No, that gives typescript error.

```ts
No overload matches this call.
  The last overload gave the following error.
    Argument of type 'number' is not assignable to parameter of type 'ValueWithSubQueries<string>'.
```

With `select('1')` the generated query is different.

Reply:
Ahh. Which means we need to fix the typescript types. Can you test it ignoring the typescript error and see if works fine at runtime?

Reply:
Yes, it works fine at runtime. Should I make the change and push? or wait for the typescript type fix first.

Reply:
yup. you can push the code. I'll fix types later

Reply:
Done.